pkgs/by-name/az/azure-cli/extensions-generated.nix

```diff
@@ -6,6 +6,13 @@
 #   ./query-extension-index.sh --requirements=false --download --nix --cli-version=<version>
 { mkAzExtension }:
 {
+  acat = mkAzExtension rec {
```

Comment:
Please commit each extension init/update in a separate commit, following the commit naming conventions.